build.sbt

```diff
@@ -17,7 +33,37 @@ val commonSettings: Seq[Setting[_]] = Seq(
       url("https://github.com/scala-js/scala-js-env-selenium"),
       "scm:git:git@github.com:scala-js/scala-js-env-selenium.git",
       Some("scm:git:git@github.com:scala-js/scala-js-env-selenium.git")))
-)
+) ++ mimaDefaultSettings
+
+val previousArtifactSetting: Setting[_] = {
+  previousArtifact := {
+    val scalaV = scalaVersion.value
+    val scalaBinaryV = scalaBinaryVersion.value
+    if (!scalaVersionsUsedForPublishing.contains(scalaV)) {
+      // This artifact will not be published. Binary compatibility is irrelevant.
+      None
+    } else if (newScalaBinaryVersionsInThisRelease.contains(scalaBinaryV)) {
+      // New in this release, no binary compatibility to comply to
+      None
+    } else {
+      val thisProjectID = projectID.value
+      val previousCrossVersion = thisProjectID.crossVersion match {
+        case ScalaJSCrossVersion.binary => previousBinaryCrossVersion
```

Comment:
This is a bit overkill for this project. It does not contain any Scala.js project, so `ScalaJSCrossVersion.binary` never happens. So this `match` is unnecessary, and so is `previousBinaryCrossVersion`.